feature-libs/cart/import-export/components/export-entries/export-entries.service.ts

```diff
@@ -23,13 +28,16 @@ export class ExportEntriesService {
     protected activeCartService: ActiveCartService,
     protected savedCartDetailsService: SavedCartDetailsService,
     protected importExportConfig: ImportExportConfig,
-    protected translationService: TranslationService
+    protected translationService: TranslationService,
+    protected globalMessageService: GlobalMessageService,
+    protected exportCsvService: ExportCsvService
   ) {}
 
-  private get additionalColumns(): ExportColumn[] {
-    return (
-      this.importExportConfig.cartImportExport?.export?.additionalColumns ?? []
-    );
+  private get exportConfig(): ExportConfig {
+    return {
+      ...this.importExportConfig.cartImportExport?.export,
+      ...defaultImportExportConfig.cartImportExport?.export,
```

Comment:
should that not default one overridden by customer-specified? Now it seems vice-versa

Reply:
btw for global configuration - and this is a global configuration, we don't do that. If we provide default config already, it's easy to override only part of that.